app/controllers/web/admin/news_controller.rb

```diff
@@ -1,11 +1,11 @@
 class Web::Admin::NewsController < Web::Admin::ApplicationController
   def index
-    @published_news = NewsDecorator.decorate_collection News.published.order('published_at DESC') #FIXME using another decorator in that file
+    @published_news = NewsDecorator.decorate_collection News.published.order('published_at DESC') 
```

Comment:
Line is too long. [98/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.